.vscode/settings.json

```diff
@@ -1,6 +1 @@
-{
```

Comment:
Are you sure we want to remove this for others? I found it quite helpful in reducing on used imports. I know it gave you an issue but maybe there's something else going on that we can sync on Tues to figure out?

Reply:
That makes a lot of sense.  I am curious as to how to keep the react imports while using this script.  The react imports are required to take care of the lint errors, correct? 

Am interested to discuss this at the standup tomorrow.